src/components/date-picker/_date-picker.scss

```diff
@@ -0,0 +1,280 @@
+@import '../../globals/scss/colors';
+@import '../../globals/scss/css--reset';
+@import '../../globals/scss/typography';
+@import '../../globals/scss/import-once';
+@import 'flatpickr.scss';
```

Comment:
Make sure to test if styles work correctly when importing `_data-picker.scss` only